### PR TITLE
Applied ADOC formatting and content updates to FCS_DTLSS_EXT.1 and in Appendix C.

### PR DESCRIPTION
**Pull request by mclearn**
Initial update to FCS_DTLSS_EXT.1 and appendix C material.

Also made one minor update to FCS_TLSS_EXT.1.5 to sync with FCS_DTLSS_EXT.1.8 to accommodate (what I think is) a better grammar update.

### CODE REVIEW

File: NDcPP_v2_2e.adoc [hunk 1]
```diff
@@ -2333,61 +2333,116 @@ _The use of the early data extension and the post-handshake client authenticatio
...
+
+*_Application Note {counter:appnote_count}_*
+
+_The option 'provides' shall be selected if the TOE allows configuration of the list of ciphers as defined in FCS_DTLSS_EXT.1.1 (e.g. enabling/disabling of ciphers, ordering, assigning priorities). Otherwise the option 'does not provide' shall be selected._
+
+*FCS_DTLSS_EXT.1.9* The TSF shall prohibit the use of the following extensions: [
+
+* [.underline]#Early data extension#
+
+].
```
Comment (kr15tyk) [inline NDcPP_v2_2e.adoc:2434]: @mclearn brackets and underlined text are not included in the MS Word doc.

File: NDcPP_v2_2e.adoc [hunk 2]
```diff
@@ -3603,24 +3658,42 @@ FIA_X509_EXT.1 X.509 Certificate Validation
 FIA_X509_EXT.2 X.509 Certificate Authentication
 ____
 
-*FCS_DTLSS_EXT.1.1* The TSF shall implement [selection: _DTLS 1.2 (RFC 6347), DTLS 1.0 (RFC 4347)_] supporting the following ciphersuites:
+*FCS_DTLSS_EXT.1.1* The TSF shall implement [selection: _DTLS 1.3 (RFC tbd), DTLS 1.2 (RFC 6347)_] and [selection: _DTLS 1.0 (RFC 4347), no additional DTLS version_] supporting the following ciphersuites:
```
Comment (kr15tyk) [inline NDcPP_v2_2e.adoc:3661]: @mclearn Same as previous comment...
Per the MS Word doc v13d, the closing bracket should be after 'DTLS 1.0 (RFC 4347)'.

File: NDcPP_v2_2e.adoc [hunk 3]
```diff
@@ -3603,24 +3658,42 @@ FIA_X509_EXT.1 X.509 Certificate Validation
...
 * DTLS records previously received.
 * DTLS Records too old to fit in the sliding window.
 
-*FCS_DTLSS_EXT.1.7* The TSF shall support [selection: _no session resumption or session tickets, session resumption based on session IDs according to RFC 4346 (TLS1.1) or RFC 5246 (TLS1.2), session resumption based on session tickets according to RFC 5077_].
+
+*FCS_DTLSS_EXT.1.7* The TSF shall support [selection: _no session resumption or session tickets, session resumption based on session IDs according to RFC4346 (TLS1.1) or RFC5246 (TLS1.2), session resumption based on session tickets according to RFC5077 (TLS1.2), session resumption according to RFC8446 (TLS1.3)_].
+
+*FCS_DTLSS_EXT.1.8* The TSF [selection: _provides, does not provide_] the ability to configure the list of supported ciphersuites as defined in FCS_DTLSS_EXT.1.1.
+
+*FCS_DTLSS_EXT.1.9* The TSF shall prohibit the use of the following extensions: [assignment: [.underline]#list of prohibited DTLS extensions#].
```
Comment (kr15tyk) [inline NDcPP_v2_2e.adoc:3694]: @mclearn This SFR is missing the bullet item, and same comment as before... 'brackets and underlined text are not included in the MS Word doc'.

**Review by kr15tyk — CHANGES_REQUESTED** [review]

File: NDcPP_v2_2e.adoc [hunk 4]
```diff
@@ -2333,61 +2333,116 @@ _The use of the early data extension and the post-handshake client authenticatio
 
 _The use of out-of-band provisioned pre-shared keys creates a potential security concern since the entropy used to generate the PSK is outside of the control of the TOE. PSKs are used in DTLS 1.3 to provide session resumption; however, in this case, the TSF is responsible for generating appropriately strong keying material.  Use of (EC)DHE further enhances security when using PSKs by providing forward secrecy._
 
-*FCS_DTLSS_EXT.1 DTLS Server Protocol Without Mutual Authentication*
+*FCS_DTLSS_EXT.1 DTLS Server Protocol*
 
-*FCS_DTLSS_EXT.1.1* The TSF shall implement [selection: _DTLS 1.2 (RFC 6347), DTLS 1.0 (RFC 4347)_] supporting the following ciphersuites:
+*FCS_DTLSS_EXT.1.1* The TSF shall implement [selection: _DTLS 1.3 (RFC tbd), DTLS 1.2 (RFC 6347)_] and [selection: _DTLS 1.0 (RFC 4347), no additional DTLS version_] supporting the following ciphersuites:
```
[the remark this answers is not in the document]
Comment (mclearn) [reply NDcPP_v2_2e.adoc:2338]: If v1.0 is the only thing in the selection, then it must be picked by the rules of CC selections.  If you want to ensure that v1.0 is an optional selection, then you need to either put "none" or, as I chose here, pull "no additional DTLS version" in as a possible selection completion.

File: NDcPP_v2_2e.adoc (hunk 2 above)
Comment (mclearn) [reply NDcPP_v2_2e.adoc:3661]: See response above.

File: NDcPP_v2_2e.adoc (hunk 1 above)
Comment (mclearn) [reply NDcPP_v2_2e.adoc:2436]: I reviewed section 6.1 of the NDcPP and I think I can get away with removing the brackets and applying italicized formatting here.

File: NDcPP_v2_2e.adoc (hunk 3 above)
Comment (mclearn) [reply NDcPP_v2_2e.adoc]: Reviewed formatting options and altered to be a simple assignment without underlining.

File: NDcPP_v2_2e.adoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (kr15tyk) [reply NDcPP_v2_2e.adoc:2338]: Agreed

File: NDcPP_v2_2e.adoc (hunk 2 above)
Comment (kr15tyk) [reply NDcPP_v2_2e.adoc:3662]: agreed

File: NDcPP_v2_2e.adoc (hunk 1 above)
Comment (kr15tyk) [reply NDcPP_v2_2e.adoc]: @mclearn I agree with removing the brackets and applying italicized formatting.

File: NDcPP_v2_2e.adoc (hunk 3 above)
Comment (mclearn) [reply NDcPP_v2_2e.adoc]: I think this can be resolved based on the associated comment earlier on agreeing to italics.

Comment (kr15tyk) [reply NDcPP_v2_2e.adoc]: Agreed

**Review by kr15tyk — APPROVED** [review]

**Pull request merged**